src/status_im/translations/uk.cljs

```diff
@@ -53,255 +71,294 @@
-   :more                                  "ще"
-   :cancel                                "Відмінити"
-   :no-statuses-found                     "Статуси не знайдено"
-   :swow-qr                               "Показати QR"
-   :browsing-open-in-web-browser          "Відкрити у браузері"
-   :delete-group-prompt                   "Це не вплине на контакти"
-   :edit-profile                          "Редагувати профіль"
-
-   :enter-password-transactions
-                                          {:one   "Підтвердіть операцію, ввівши пароль"
-                                           :other "Підтвердіть операції, ввівши пароль"}
-   :unsigned-transactions                 "Непідписані транзакції"
-   :empty-topic                           "Тема пуста"
-   :to                                    "До"
-   :group-members                         "Члени групи"
-   :estimated-fee                         "Прибл. комісія"
-   :data                                  "Дані"})
\ No newline at end of file
+   ;; Wallet Send
+   :wallet-choose-recipient               "Вибрати Одержувача"
+   :wallet-choose-from-contacts           "Вибрати зі Списку Контактів"
+   :wallet-address-from-clipboard         "Використовувати скопійовану раніше адресу"
+   :wallet-invalid-address                "Невірна адреса: \n {{data}}"
+   :wallet-browse-photos                  "Перегляд Фото"
+   :validation-amount-invalid-number      "Сума не є дійсним номером"
+   :validation-amount-is-too-precise      "Сума занадто точна. Найменша одиниця, яку ви можете надіслати, є 1 Wei (1x10^-18 ETH)"
+
+   ;; network settings
+   :new-network                           "Нова мережа"
+   :add-network                           "Додати мережу"
+   :add-new-network                       "Додати нову мережу"
+   :existing-networks                     "Існуючі мережі"
+   :add-json-file                         "Додати JSON файл"
+   :paste-json-as-text                    "Вставити JSON файл"
+   :paste-json                            "Вставити JSON"
+   :specify-rpc-url                       "Зазначити RPC URL"
+   :edit-network-config                   "Редагувати конфігурацію мережі"
+   :connected                             "Підключено"
+   :process-json                          "Обробити JSON"
+   :error-processing-json                 "Помилка обробки JSON"
+   :rpc-url                               "RPC URL"
+   :remove-network                        "Видалити мережу"
+   :network-settings                      "Налаштування мережі"
+   :edit-network-warning                  "Обережно, редагування мережевих даних може вимкнути цю мережу"
+   :connecting-requires-login             "Підключення до іншої мережі вимагає входу в систему"
+   :close-app-title                       "Увага!"
+   :close-app-content                     "Програма зупиниться та закриється. Коли ви знову відкриєте його, вибрана мережа буде використовуватися"
```

Comment:
```
Додаток буде зупинено і закрито. Обрана мережа буде використана під час наступного запуску. 
```
Not 100% sure though,  but this version sounds more natural imo.

Reply:
Agree.